src/status_im2/contexts/wallet/create_account/select_keypair/view.cljs

```diff
@@ -8,6 +8,24 @@
     [utils.i18n :as i18n]
     [utils.re-frame :as rf]))
 
+(defn keypair-options
```

Comment:
maybe we should put this in a separate options file? 
e.g `src/status_im2/contexts/wallet/create_account/select_keypair/options.cljs`

Reply:
@J-Son89 I don't think we should be rendering views in files named other than `view`. 
I think this is a small component to put in a separate namespace, and is not reused in other screens.